module/deltagreen.js

```diff
@@ -197,3 +199,18 @@ Hooks.on("renderGamePause", function (_, html, options) {
     html.querySelector("img").classList.remove("fa-spin"); // I don't like the logo spinning personally
   } catch {}
 });
+
+Hooks.once("setup", async () => {
+  document.addEventListener("click", (event) => {
```

Comment:
We should use the hook `renderChatLog` instead of `setup`. that way we can have a more specific listener, rather than listening on the whole document. See https://foundryvtt.com/api/functions/hookEvents.renderApplicationV2.html

```suggestion
Hooks.once("renderChatLog", async (app, element, context, options) => {
  element.addEventListener("click", (event) => {
```